src/Sylius/Bundle/RbacBundle/spec/Provider/SecurityIdentityProviderSpec.php

```diff
@@ -52,6 +53,13 @@ function it_returns_null_if_token_exists_but_still_no_authenticated_user($securi
         $this->getIdentity()->shouldReturn(null);
     }
 
+    function it_returns_null_if_token_exists_but_its_an_anonymous_user($securityContext, AnonymousToken $token)
+    {
+        $securityContext->getToken()->shouldBeCalled()->willReturn($token);
```

Comment:
Because `$securityContext->getToken()->willReturn($token);` is allways called (no need to check that). We want to check that the method `getIdentity()` returns null when the token is an `AnonymousToken`. That's all! :) Are you ok ?